server/chat-plugins/datasearch.js

```diff
@@ -1212,12 +1255,29 @@ function runMovesearch(target, cmd, canAll, message) {
 	}
 	if (results.length > 1) {
 		results.sort();
+		if (sort) {
+			let prop = sort.slice(0, -1);
+			let direction = sort.slice(-1);
+			results.sort((a, b) => {
+				let move1prop = dex[toId(direction === '+' ? a : b)][prop], move2prop = dex[toId(direction === '+' ? b : a)][prop];
+				if (typeof move1prop === 'boolean') {
+					move1prop = ~~move1prop;
+				}
+				if (typeof move2prop === 'boolean') {
+					move2prop = ~~move2prop;
+				}
+				return move1prop - move2prop; // Incase they are boolean valuess, they are converted to 0 or 1
```

Comment:
```suggestion
				return move1prop - move2prop;
```

(bad place for comment)

Reply:
```suggestion
				return (move1prop - move2prop) * (direction === '+' ? 1 : -1);
```